docs/faq.rst

```diff
@@ -99,6 +99,18 @@ support multiple versions of Python. The Python 3.4 ``abi3`` wheel can be used
 with any version of Python greater than or equal to 3.4. Recent versions of
 ``pip`` will automatically install ``abi3`` wheels.
 
+``ModuleNotFoundError``: No module named ``'idna'``
```

Comment:
Instead of, or in addition to, this FAQ, please make the lazy `import idna` raise a better error message on `ImportError`